crates/bindings-csharp/Codegen/Module.cs

```diff
@@ -225,7 +225,7 @@ public void Initialize(IncrementalGeneratorInitializationContext context)
 
                             public void Insert() {{
                                 var bytes = SpacetimeDB.Runtime.Insert(tableId.Value, this);
-                                // bytes should contain modified value now with autoinc fields updated
+                                // bytes should contain modified value now with auto_inc fields updated
```

Comment:
Is this a search-replace change? Probably shouldn't be necessary here and in the snapshots.